.github/mergify.yml

```diff
@@ -20,7 +20,36 @@ pull_request_rules:
       - label=status:merge-when-green
     actions:
       queue:
-        method: merge
+        merge_method: merge
+        update_method: rebase
```

Comment:
I think update method should be `rebase` by default for us.

Reply:
yes